### PR TITLE
Migration guide: clarify the newest blacklisting/whitelisting API

### PR DESCRIPTION
**Pull request by kyrylo**
I was changed since https://github.com/airbrake/airbrake/pull/530.

### CODE REVIEW

**Comment by shifi** [comment]
LGTM
We should also [update relevant wiki page](https://github.com/airbrake/airbrake/wiki/Filtering-sensitive-data).

File: docs/Migration_guide_from_v4_to_v5.md [hunk 1]
```diff
@@ -314,13 +314,24 @@ Instead of [various filter options](#params-filters) the library supports
...
 # New way
+Airbrake.configure do |c|
+  c.blacklist_keys << [:credit_card_number]
+end
+```
+
+###### Airbrake v5.0.0-v5.1.0 API (deprecated)
+
+```ruby
+# New way
```
Comment (shifi) [inline docs/Migration_guide_from_v4_to_v5.md]: Maybe remove comment `# New way` since it's deprecated?

**Comment by kyrylo** [comment]
> We should also update relevant wiki page.

Are you sure? The wiki was deprecated entirely with the v5 release.

**Comment by shifi** [comment]
> Are you sure? The wiki was deprecated entirely with the v5 release.

Oh, I forgot about that. Nvm then. Thanks.

File: docs/Migration_guide_from_v4_to_v5.md (hunk 1 above)
Comment (kyrylo) [inline docs/Migration_guide_from_v4_to_v5.md]: Sounds good, fixed.

**Pull request merged**